DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -62,8 +65,16 @@ FROM Tutorials
                             )
                             AND NextCustomisationTutorials.TutorialID = NextTutorials.TutorialID
 
+                         LEFT JOIN CustomisationTutorials AS NextSectionCustomisationTutorials

```

Comment:
Why do we need a new join on CustomisationTutorials instead of a adding to the one we have? 🙂

Reply:
The previous join only selected `Status = 1` tutorials; tutorials whose pages can be visited for the `Next tutorial` button.

This join used to find the next section has a slightly different purpose: to find sections that either have tutorials that can be viewed (`Status = 1`), have a diagnostic assessment (`DiagStatus = 1`) or have a post learning assessment (`IsAssessed = 1`). In each of these cases the section page can be visited (and if none are true then they can be skipped). I don't think it is possible to cover all of these cases with just one join, but I might be missing something 🙂

Reply:
Okay, fair enough 🙂